hscommon/hsenum/cof_editor_state.go

```diff
@@ -0,0 +1,12 @@
+package hsenum
+
+// COFEditorState represents cof editor's states
+type COFEditorState int
+
+// cof editor's states
+const (
+	COFEditorStateViewer COFEditorState = iota
+	COFEditorStateAddLayer
+	COFEditorStateAddDirection
+	COFEditorStateConfirm
+)
```

Comment:
I dont think we need to put _every_ enumeration into `hsenum`, only the enums we want to re-use. Only the COF editor/viewer uses these states, I would imagine that these would be defined in `hswindow/hseditor/hscofeditor/` and not be exported.